### DISCUSSION TITLE
我是3月24买的会员想要你们官方截图或发票来报销 谢谢

### DISCUSSION THREAD

**Discussion by jacktt6254** [getlantern/lantern-forum-cn#500] [Q&A]
我是3月24买的会员想要你们官方截图或发票来报销 谢谢

**Comment by jacktt6254** [comment]
709638505@qq.com用这个电邮注册的，麻烦发到这个电邮

**Reply by MaxMKMax** [reply]
@jacktt6254 您好. 您查看邮箱或垃圾邮箱里有发票吗? 一般会自动发到您的邮箱的. 
如果您没找到那再来找我. 我看支付团队能给您找一下吗.
